### ISSUE TITLE
two votes

### ISSUE THREAD

**Issue by etorfa** [emoreno911/Notitunes#2]
i found that is possible vote is you vote, but is change the vote.... is possible detect tht you already vote?

**Comment by emoreno911** [comment]
Yes, I missed that verification. Soon I'll be making an update to fix It
